data/trie/patriciaMerkleTrie.go

```diff
@@ -286,6 +286,11 @@ func (tr *patriciaMerkleTrie) String() string {
 	return writer.String()
 }
 
+// ClosePersister will close trie persister
+func (tr *patriciaMerkleTrie) ClosePersister() {
+	_ = tr.db.Close()
```

Comment:
this method returns an error. In the interface, the `ClosePersister()` is void type because in AccountsDB there is no returned error. So please change the interface's method to return an error and return it here, or ar least log here if there is an error

Reply:
better log the error here and leave the interface as it is

Reply:
error is now returned